transaction/core/src/signed_contingent_input.rs

```diff
@@ -1,17 +1,20 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
 // Copyright (c) 2022 The MobileCoin Foundation
```

Comment:
Nit: Duplicate copyright
```suggestion
// Copyright (c) 2018-2022 The MobileCoin Foundation
```